web/index.html

```diff
@@ -181,6 +181,7 @@
   left: 0;
   height: calc(100% - 60px);
   width: 90%;
+  overflow-x: clip;
```

Comment:
This fixes an annoying issue where long filenames drew text over the bar code. :scream: 

(ecc made the transfer work anyway :tada: )